portfolio/src/main/webapp/script.js

```diff
@@ -93,4 +98,24 @@ function createCommentDiv(commentObj) {
   // TODO: Convert from fairly useless MS time to hour/minute.
 
   return commentDiv;
-}
\ No newline at end of file
+}
+
+function loadMap() {
+  // Create the script tag, set the appropriate attributes
```

Comment:
This comment isn't conveying anything that I couldn't already understand by reading your code. In general, your code should speak for itself, which is where good variable and function names comes in. You should only keep comments for function descriptions or for explaining WHY a certain line/block of code is needed in case it's not intuitive. Let me know if you would like more examples of this

Reply:
Sounds good! This snippet is more or less directly from the Google API documentation, which is probably why the comments are a bit over-explanatory. I should have taken them out.